SkillSwapHub/src/main/resources/static/js/registerUser.js

```
@@ -39,19 +44,49 @@ async function submitRegistrationForm(event) {
-        let errorMessage = `<p class="validation-error">${errors[key]}</p>`;
-        messageBox.innerHTML += errorMessage;
+
+        showMessage(errors[key], "error");
     }
 }
 
 //Function to display general messages
 function showMessage(message, type) {
-    const messageDiv = document.getElementById("message");
-    messageDiv.innerHTML = `<p style="color: ${type === "success" ? "#0F5133" : "#84202A"}; font-size: 10px;" class="submission-msg">${message}</p>`;
+    let toastContainer = document.getElementById("toast-container");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'toastContainer' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)